README.md

```diff
@@ -347,3 +390,19 @@ Returns a success message.
 | expanded     | boolean       |                |
 | acknowledged | boolean       |                |
 | refId        | string        | uuid &#124; '' |
+
+### Mnemonic
+
+| Property       | Type   | Description |
+| -------------- | ------ | ----------- |
+| id             | string | uuid        |
+| mnemonicId     | string |             |
+| status         | Status |             |
+| unit           | string |             |
+| thresholdMax   | number |             |
+| thresholdMin   | number |             |
+| currentValue   | number |             |
+| subsystem      | string |             |
+| childSubsystem | string |             |
+| measurement    | string |             |
+| contactRefId   | string |             |
```

Comment:
I think we need to update the Alert schema to replace refId with contactRefId also.